src/errors/validation_exception.rs

```diff
@@ -113,12 +113,12 @@ static URL_PREFIX: GILOnceCell<String> = GILOnceCell::new();
 
 fn get_url_prefix(py: Python, include_url: bool) -> Option<&str> {
     if include_url {
-        Some(URL_PREFIX.get_or_init(py, || {
-            format!(
-                "https://errors.pydantic.dev/{}/v/",
-                get_pydantic_version(py).unwrap_or("latest")
-            )
-        }))
+        let pydantic_version = match get_pydantic_version(py) {
```

Comment:
we need to do this inside the `get_or_init` to avoid the overhead of calling this each time, best to create a new function including this logic and the `format!`